components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -726,7 +726,7 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
             this.textboxRef.current.blur(); // Fixes Firefox bug which causes keyboard shortcuts to be ignored (MM-22482)
             return;
         }
-        if (this.textboxRef.current && (keepFocus || !UserAgent.isMobile())) {
+        if (this.textboxRef.current && keepFocus) {
```

Comment:
@ismaaylSpiria I think we need this.

Reply:
just this one? the other 3 can be removed?

Reply:
@AshishDhama I tested on Firefox (Mac) and Safari (iOS), and this PR works fine in its current state. Are you sure this is needed?

Reply:
@hmhealey Yeah, I think because @ismaaylSpiria have reverted the change from `advanced_create_post.tsx`, @ismaaylSpiria sorry for missing this earlier we need to revert the same change from `create_post.tsx` as well, In short we just need to remove `!UserAgent.isMobile()` from `advanced_create_commet.tsx` and `create_comment.tsx`